corehq/apps/user_importer/importer.py

```diff
@@ -789,8 +789,7 @@ def remove_web_user_from_domain(domain, user, username, upload_user, user_change
             if user_change_logger:
                 user_change_logger.add_info(UserChangeMessage.invitation_revoked_for_domain(domain))
         else:
-            raise UserUploadError(_("You cannot remove a web user that is not a member of this project."
-                                    " {web_user} is not a member.").format(web_user=user))
+            raise UserUploadError(_("You cannot remove a web user that is not a member of this project."))
```

Comment:
I updated the message and removed web_user, because it would either currently be
1. `None`
2. str representation of WebUser `WebUser <web-user-couch-id>`
Both of which were not useful. In the upload file web user's username is passed in the upload, so the user doing the upload knows which user this message is corresponding to. It would be nice to separate out missing and non-member messages but I pulled back on that to avoid changing it too much,